examples/model_compress/MeanActivation_torch_cifar10.py

```diff
@@ -82,7 +83,8 @@ def main():
 
     # Test base model accuracy
     print('=' * 10 + 'Test on the original model' + '=' * 10)
-    model.load_state_dict(torch.load('vgg16_cifar10.pth'))
+    if os.path.isfile('vgg16_cifar10.pth'):
+        model.load_state_dict(torch.load('vgg16_cifar10.pth'))
```

Comment:
It seems the `load_state_dict` and the retrain part should swap order, current logic is strange.

Reply:
I don't understand why is the logic strange. Could you be more specific?

Reply:
if args.retrain is True,  we do not need to load model here.
if args.retrain is False, we  got a model with random parameter before `load_state_dict`,  our pruner should not prune a random model, so we should assert the 'vgg16_cifar10.pth' must exist.

Reply:
fixed